### PR TITLE
docs:improved Environment Variables guide

### LINKED ISSUE #235: Improve Environment Variables guide
**Is your feature request related to a problem? Please describe.**
I just think it'd be helpful for all of us to have a full description on how to setup dev environment for this project. 

**Describe the solution you'd like**
I'd like to see something like this (below) included in [the guide](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/environment-variables.md) at `/docs/environment-variables.md`

> To properly set up `dev` environment, an **email address is required**. We recommend creating a temporary email (e.g.: Gmail, Mail.com, etc)

Also, I think that the current description of `MAIL_DEFAULT_SENDER` and `APP_MAIL_USERNAME` is insufficient. I'd change it to:

> `MAIL_DEFAULT_SENDER` Email used to send email verification emails. This has to be a real email address that you've got access to. This is in string format.

> `APP_MAIL_USERNAME` Username of the email account used to send verification emails. It must be equal to part of the email address before the `@` sign. For example, if your email address is `johnclark@gmail.com`, this will be `johnclark`. This is in string format.

**Additional context**

Conversation from which this info comes: https://systers-opensource.zulipchat.com/#narrow/stream/212722-gci/topic/Mentorship.20Backend.20Setup

### PR DESCRIPTION
**Pull request by vj-codes**
### Description
Improved environment variables guide .

Fixes #235

### Type of Change:

- Documentation

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by rpattath — APPROVED** [review]
LGTM for OSH

**Review by aditmehta9 — APPROVED** [review]

**Comment by aditmehta9** [comment]
After Changes:
image

File: docs/environment-variables.md [hunk 1]
```diff
@@ -43,13 +43,13 @@ Email verification is when a user registers into the application and to be able
 
 | Environment Variable | Description                                                                                                     | Example                |
 |----------------------|-----------------------------------------------------------------------------------------------------------------|------------------------|
-| MAIL_DEFAULT_SENDER  | Email used to send email verification emails. This is in string format.                                         | 'some_username@gmail.com' |
+| MAIL_DEFAULT_SENDER  |Email used to send email verification emails. This has to be a real email address that you've got access to. For example , johnclark@gmail.com . This is in string format.                                       | 'some_username@gmail.com' |
```
Comment (devkapilbansal) [inline docs/environment-variables.md:46]: @vj-codes there is already an example tab here, then should we include the example in description too?

**Review by devkapilbansal — CHANGES_REQUESTED** [review]

File: docs/environment-variables.md (hunk 1 above)
Comment (vj-codes) [reply docs/environment-variables.md:46]: @devkapilbansal the issue suggested adding the example in the description as well just to make things clearer , what say?

Comment (vj-codes) [reply docs/environment-variables.md:46]: @devkapilbansal can you update the label do you still need to remove the example from description 👀

**Review by isabelcosta — APPROVED** [review]
Thank you @vj-codes for doing this! I think this will make the instructions clearer to newcomers :)

**Pull request merged**